src/ui/components/Library/Team/View/Tests/Overview/TestErrorList.tsx

```diff
@@ -0,0 +1,181 @@
+import { motion } from "framer-motion";
+import orderBy from "lodash/orderBy";
+import { Dispatch, SetStateAction, useState } from "react";
+import { ContextMenuItem, useContextMenu } from "use-context-menu";
+
+import Icon from "replay-next/components/Icon";
+import { TestExecution } from "shared/test-suites/TestRun";
+import LibraryDropdownTrigger from "ui/components/Library/LibraryDropdownTrigger";
+
+import { getTruncatedRelativeDate } from "../../Recordings/RecordingListItem/RecordingListItem";
+import styles from "../../../../Library.module.css";
+
+const MAX_ERRORS_SHOWN = 5;
+const MAX_REPLAYS_SHOWN = 3;
+
+export function TestErrorList({
+  errorFrequency,
+  executions,
+}: {
+  errorFrequency: Record<string, number>;
+  executions: TestExecution[];
+}) {
+  const [selectedError, setSelectedError] = useState<string | null>(null);
+  const [filterByTime, setFilterByTime] = useState<number | null>(null);
+
+  const {
+    contextMenu: contextMenuTimeFilter,
+    onContextMenu: onClickTimeFilter,
+    onKeyDown: onKeyDownTimeFilter,
+  } = useContextMenu(
+    <>
+      <ContextMenuItem disabled onSelect={() => setFilterByTime(1 / 24)}>
+        Last hour
+      </ContextMenuItem>
+      <ContextMenuItem disabled onSelect={() => setFilterByTime(1)}>
+        Last day
+      </ContextMenuItem>
+      <ContextMenuItem disabled onSelect={() => setFilterByTime(7)}>
+        Last week
+      </ContextMenuItem>
+      <ContextMenuItem disabled onSelect={() => setFilterByTime(30)}>
+        Last month
+      </ContextMenuItem>
+      <ContextMenuItem onSelect={() => setFilterByTime(null)}>All time</ContextMenuItem>
```

Comment:
There's something that feels off to me about showing time filters for both the outer test runs column _and_ the inner top errors column. For example, filtering the outer test runs column to show only failed tests for the past hour and then filtering top errors for the selected test to be for "all time" feels misleading and confusing. Seems like the inner column should mirror the outer setting?

cc @jonbell-lot23 because I realize this is more of a UX comment.

Reply:
Yeah this is a tough one. The middle mirroring the left can cause other issues. I agree that this can be confusing, but sometimes this granularity is needed and I think the alternative is worse. Will be keeping an eye on this for sure.

Reply:
@jonbell-lot23 for that one, in which cases do you imagine the second dropdown will be used?

The more I think about this the less sure I am that we need the second time dropdown since it's redundant. Any narrowing-down action that applies to the second dropdown can just as easily be applied with the first dropdown.